togglsync/jira_wrapper.py

```diff
@@ -112,8 +134,12 @@ def dictFromTogglEntry(cls, togglEntry):
     def get(self, issue_key):
         try:
             for worklog in self.jira_api.worklogs(issue_key):
-                if worklog.author.name == self.user_name:
-                    yield JiraTimeEntry.fromWorklog(worklog, issue_key)
+                try:
+                    if worklog.author.name == self.user_name:
```

Comment:
What do you think about adding sth like `JiraTimeEntry.find_username(worklog)`. Then using it both in `JiraHelper.get()` and `JiraTimeEntry.fromWorklog()` ?